src/instantsearch/instantsearch.ts

```diff
@@ -260,15 +262,14 @@ export class NgAisInstantSearch implements AfterViewInit, OnInit, OnDestroy {
     }
 
     // custom algolia client agent
-    if (!config.searchClient && !config.createAlgoliaClient) {
-      const client = algoliasearch(config.appId, config.apiKey);
-      client.addAlgoliaAgent(`angular-instantsearch ${VERSION}`);
+    config.searchClient =
```

Comment:
The [storybook breaks](https://deploy-preview-481--angular-instantsearch.netlify.com/examples/storybook/?selectedKind=ClearRefinements&selectedStory=default&full=0&addons=1&stories=1&panelRight=0) because we need to remove `appId` and `apiKey` from the configuration before passing it to the InstantSearch.js instantiation.

If we want to release this feature before releasing v3, we actually might want to revert this change and keep the `if (typeof config.searchClient.addAlgoliaAgent === 'function')` statement. It might break some people's implementation who still rely on `createAlgoliaClient`.